content/docs/tina-cloud/connecting-site.md

```diff
@@ -68,8 +68,8 @@ import { defineSchema } from 'tinacms'
 const branch =
   process.env.NEXT_PUBLIC_TINA_BRANCH ||
   process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF ||
-  process.env.HEAD ||
-  ''
```

Comment:
So the reason I had this set as an empty string as a fallback, is because `branch` is currently required, however, the environment variables may be undefined. (I believe our client generation still catches this and throws an error on empty string).

Otherwise, if you're using typescript, you need to do: something like: 
```
   branch:
      process.env.NEXT_PUBLIC_TINA_BRANCH! || 
      process.env.NEXT_PUBLIC_VERCEL_GIT_COMMIT_REF! || 
      process.env.HEAD!
```
which doesn't work on `.js`.

Open to ideas on a best practice for setting this branch variable though!

Reply:
Makes sense! I will add the `|| ''` back in